ckanext/data_qld/auth_functions.py

```diff
@@ -1,29 +0,0 @@
-# encoding: utf-8
-
-import ckan.authz as authz
-from ckantoolkit import _, auth_allow_anonymous_access
-
-
-@auth_allow_anonymous_access
-def has_user_permission_for_some_org(context, data_dict):
```

Comment:
did we find alternatives for this?

Reply:
It's present in ckanext-resource-visibility, `plugin.py` now points there.